bluesky/callbacks/best_effort.py

```diff
@@ -92,7 +92,7 @@ def start(self, doc):
 
         # Prepare a guess about the dimensions (independent variables) in case
         # we need it.
-        motors = self._start_doc.get('motors') or None
+        motors = self._start_doc.get('motors')
```

Comment:
This changes behavior in the case when the `get` returns an empty list, not sure of the consequence of that though....

Reply:
hmm that's interesting. I'm more in favor of a consistent result that uncertain. Tests pass but I think this PR should be closed. What do yo think?